test/vm/flatpak_fedora_repo_disabled.yml

```
@@ -10,7 +10,9 @@
     register: services_state
 
   - name: Check that flatpak-add-fedora-repos is disabled
+    when: services_state['ansible_facts']['services']['flatpak-add-fedora-repos.service'] is defined
     ansible.builtin.assert:
       that:
         - services_state['ansible_facts']['services']['flatpak-add-fedora-repos.service']['status'] == 'disabled'
-      fail_msg: 'flatpak-add-fedora-repos.service is not disabled'
\ No newline at end of file
+      fail_msg: 'flatpak-add-fedora-repos.service is not disabled'
+      success_msg: 'flatpak-add-fedora-repos.service is correctly disabled'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)